README.mkdn

```diff
@@ -67,13 +67,9 @@ You will need:
   when you first try to build.
 
 - `openocd` (ideally 0.11) or (if using the LPC55) `pyocd` (0.27 or later).
-  Note that the 0.10 release of OpenOCD predates the STLink v3. People are using
-  various post-0.10, pre-0.11 builds provided by system package managers, with
-  some success, but if your system isn't packaging 0.11 yet, pester them. If
-  you're going to use Homebrew on macOS to install OpenOCD, you need to use
-  `brew install --head openocd` to build the tip of the main branch rather than
-  using the latest binary release. If you need to build from source, you can
-  find the [OpenOCD v0.11.0 here](https://sourceforge.net/projects/openocd/files/openocd/0.11.0/).
+  Note that the 0.10 release of OpenOCD predates the STLink v3. People are
+  using various post-0.10, pre-0.11 builds provided by system package managers,   with some success, but if your system isn't packaging 0.11 yet, pester them.
```

Comment:
Please wrap lines at 80 columns

Reply:
Ah sorry, missed that one.  Fixed and remerged.